integration-tests/tests/hacbs/add-component-to-application-via-hacbs-tab.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { AddComponentPage } from "../../support/pages/AddComponentPage";
+import { ApplicationDetailPage } from "../../support/pages/ApplicationDetailPage";
+import { ComponentPage } from "../../support/pages/ComponentsPage";
+import { OverviewTabPage } from "../../support/pages/hacbs/OverviewTabPage";
+import { Applications } from "../../utils/Applications";
+import { Common } from "../../utils/Common";
+import { ComponentsTabPage } from "../../support/pages/hacbs/ComponentsTabPage";
+import { actionsDropdown } from "../../support/pageObjects/hacbs-po";
+
+
+describe('', () => {
```

Comment:
fill in the title of the test case please

Reply:
I'll be more cautious next time, thanks for the catch!